resources/views/layouts/app.blade.php

```diff
@@ -1,5 +1,7 @@
 {{-- Template from https://templates.digizu.co.uk/ --}}<!DOCTYPE html>
-@php $fullPageTitle = (isset($pageTitle) ? "{$pageTitle} | " : '') .  __('Onramp to Laravel'); @endphp
+@php
```

Comment:
I don't need this change to be implemented :) I'm ok with it in the former syntax